manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "command-ask",
+  "description": "A highly context aware organization integrated chatbot",
+  "ubiquity:listeners": ["issue_comment.created"]
```

Comment:
It is missing the command definition, how can it run?

Reply:
see [this](https://github.com/ubiquity-os-marketplace/command-ask/pull/1#issuecomment-2375667319) comment but `plugin.ts` parses the comment and skips it if the bot is not mentioned. It doesn't need an entry here to actually run it just needs the payload.

Reply:
So how is that plugin used? I thought it would be something like `/ask`

Reply:
`@UbiquityOS ...` or `@ubiquityos ...`. Partner configurable right now, unsure if it should be hardcoded or not as we tend not to with bot names

Reply:
I see. Any reason to do it that way instead of a command?

Reply:
It's more natural to use it this way